docs/application/native/guides/connectivity/overview.md

```diff
@@ -29,6 +29,9 @@ You can use the following connectivity and wireless features in your native appl
 
   You can setup SoftAP for local area connections without Internet.
 
+- [Intelligent Network Monitoring (INM)](inm.md)
+
+  You can get information about the network provided by Linux functions. You can detect IP collisions, dump network status, dump TCP, monitor ethernet, monitor Wi-Fi module state, and get network statistics.
```

Comment:
@suraj-geddam A small tweak in the sentence. Just add a coma after "network" as:

You can get information about the network, provided by Linux functions.

Reply:
Yes, done